fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransaction.java

```diff
@@ -307,7 +307,8 @@ public static LoanTransaction copyTransactionProperties(final LoanTransaction lo
                 loanTransaction.feeChargesPortion, loanTransaction.penaltyChargesPortion, loanTransaction.overPaymentPortion,
                 loanTransaction.reversed, loanTransaction.paymentDetail, loanTransaction.externalId);
 
-        if (LoanTransactionType.CHARGE_PAYMENT.equals(loanTransaction.getTypeOf())) {
+        if (LoanTransactionType.CHARGE_PAYMENT.equals(loanTransaction.getTypeOf())
+                || LoanTransactionType.REPAYMENT_AT_DISBURSEMENT.equals(loanTransaction.getTypeOf())) {
```

Comment:
We are not reverse-replay Repayment at disbursement transactions.. These never changes...